daemon/containerio/cio.go

```diff
@@ -1,37 +1,16 @@
 package containerio
 
 import (
-	"context"
-	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"sync"
-	"syscall"
 
 	"github.com/containerd/containerd/cio"
-	"github.com/containerd/fifo"
 	"github.com/sirupsen/logrus"
 )
 
-// CioFIFOSet holds the fifo pipe for containerd shim.
-//
-// It will be replaced by the containerd@v1.2
-type CioFIFOSet struct {
-	cio.Config
-	closeFn func() error
-}
-
-// Close will remove all the relatived files.
-func (c *CioFIFOSet) Close() error {
-	if c.closeFn != nil {
-		return c.closeFn()
-	}
-	return nil
-}
-
-// NewCioFIFOSet prepares fifo files.
-func NewCioFIFOSet(processID string, withStdin bool, withTerminal bool) (*CioFIFOSet, error) {
+// NewFIFOSet prepares fifo files.
+func NewFIFOSet(processID string, withStdin bool, withTerminal bool) (*cio.FIFOSet, error) {
 	root := "/run/containerd/fifo"
```

Comment:
Let make it a variable ?

Reply:
it was my plan. I will do that in next pr.